aleph-zero.yaml

```diff
@@ -0,0 +1,58 @@
+specVersion: 1.0.0
+name: subquery-nova-aleph-zero
+version: 1.0.0
+runner:
+  node:
+    name: '@subql/node'
+    version: v1.9.1
+  query:
+    name: '@subql/query'
+    version: v1.5.0
+description: Nova SubQuery project
+repository: https://github.com/nova-wallet/subquery-nova
+schema:
+  file: ./schema.graphql
+network:
+  chainId: '0x70255b4d28de0fc4e1a193d7e175ad1ccef431598211c55538f1018651a0344e'
+  endpoint: wss://ws.azero.dev
+  chaintypes:
+    file: ./dist/alephZeroChaintypes.js
+dataSources:
+  - name: main
+    kind: substrate/Runtime
+    startBlock: 1
+    mapping:
+      file: ./dist/index.js
+      handlers:
+        - handler: handleHistoryElement
+          kind: substrate/CallHandler
+        - handler: handleParachainRewarded
+          kind: substrate/EventHandler
+          filter:
+            module: parachainStaking
+            method: Rewarded
+        - handler: handleTransfer
+          kind: substrate/EventHandler
+          filter:
+            module: balances
+            method: Transfer
+        - handler: handleTransferKeepAlive
+          kind: substrate/EventHandler
```

Comment:
Interesting. Looks like a redundant handler. First of all it should be a call handler because there is no "transfer_keep_alive" event

Reply:
Thank you, I've deleted it